docs/installation.md

```diff
@@ -0,0 +1,338 @@
+    ```
+    Environment variable ION_BITCOIN_CONFIG_FILE_PATH undefined, using default path ../json/testnet-bitcoin-config.json instead.
+    ```
+- If `bitcoind` core is not alredy running, start it with the command:
+```
+/Applications/bitcoin-24.0.1/bin/bitcoind -conf=/Applications/bitcoin-24.0.1/bitcoin.conf -daemon
+```
+- Run the microservice:
+```
+npm run bitcoin
+```
+- Wait while the `getaddressinfo` command is processed. This can take up to an hour because it requires scanning the entire blockchain.
+
+### Run the ION core microservice
+- Start IPFS (in a separate shell):
+```
+ipfs daemon
+```
+- Start MongoDB:
+```
+brew services start mongodb-community
+```
+- Set environment variables
+    - Edit the `package.json` file in the root of the clone ION repository. Replace the line:
+    ```
+        "core": "node dist/src/core.js",
+    ```
+    with this line:
+    ```
+        "core": "ION_CORE_CONFIG_FILE_PATH=$HOME/.ion/testnet-core-config.json ION_CORE_VERSIONING_CONFIG_FILE_PATH=$HOME/.ion/testnet-core-versioning.json node dist/src/core.js"
+    ```
+    Note that the ION install guide gives incorrect instructions in this step. It has environment variables created in the shell, but they are not passed on to the npm scripts, resulting in the error message:
+    ```
+    Environment variable ION_CORE_CONFIG_FILE_PATH undefined, using default path ../json/testnet-core-config.json instead.
+    ```
+- Run the microservice:
+```
+npm run core
+```
+Note: this will fail unless the ION Bitcoin microservice has started successfully (Step 6).
+- Wait while the microservice synchronises its database of ION transactions.
+
+#### Verify ION is working properly
+Check the following DID resolution in your browser:
+    - testnet: http://localhost:3000/identifiers/did:ion:test:EiClWZ1MnE8PHjH6y4e4nCKgtKnI1DK1foZiP61I86b6pw
+    - mainnet: http://localhost:3000/identifiers/did:ion:EiClkZMDxPKqC9c-umQfTkR8vvZ9JPhl_xLDI9Nfk38w5w
+
+If ION is working you will see a resolved DID Document in the browser.
+
+#### Shut down the ION node
+Before shutting down the computer running ION, you can stop the two microservices by hitting `CTRL+C` in the terminals in which they are running, then stop Bitcoin with the following command:
+```
+/Applications/bitcoin-24.0.1/bin/bitcoin-cli -conf=/Applications/bitcoin-24.0.1/bitcoin.conf stop
+```
+
+## ION installation on Linux
+
+These instructions are based on the steps for Linux systems in the [ION Install Guide](https://identity.foundation/ion/install-guide/). Additional steps are noted here to be carried out at various points in the ION guide.
+
+### Linux Environment Setup
+Run:
+```
+sudo apt update
+```
+before running the ION step:
+```
+sudo apt install build-essential
+```
+
+### Setting up Bitcoin Core
+If using sidetree bitcoin install script(https://github.com/decentralized-identity/sidetree/blob/master/lib/bitcoin/setup.sh), pay attention to the following:
```

Comment:
```suggestion
If using the [sidetree bitcoin install script](https://github.com/decentralized-identity/sidetree/blob/master/lib/bitcoin/setup.sh), pay attention to the following:
```
Suggestion to avoid showing the entire url, but rather create a link out of it.